### PR TITLE
Add recipe for symfony-consul-bundle

### PR DESCRIPTION
**Pull request by akondas**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by akondas** [comment]
Looks like `Someone (not the Symfony bot nor the pull request author) approved the pull request;` doesn't seem to work ... :thinking:

**Comment by akondas** [comment]
Hello @Nyholm can I do anything else to make it merged?

File: akondas/symfony-consul-bundle/0.1/config/packages/consul.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+consul:
+    service:
+        name: 'symfony-app'
+        host: 'localhost'
+        port: 8000
+    client:
+        base_uri: http://127.0.0.1:8500
```
Comment (Nyholm) [inline akondas/symfony-consul-bundle/0.1/config/packages/consul.yaml:7]: Please correct me if Im wrong. Doesnt all these values change depending on if you are on local development or in production?

If so, they should be environment variables.

Comment (akondas) [reply akondas/symfony-consul-bundle/0.1/config/packages/consul.yaml:7]: it depends on the approach, but the idea to give environment variables there is absolutely justified

Comment (Nyholm) [reply akondas/symfony-consul-bundle/0.1/config/packages/consul.yaml:7]: Could you please update the PR accordingly?

Comment (akondas) [reply akondas/symfony-consul-bundle/0.1/config/packages/consul.yaml]: Updated, I'm sorry that it took so long.

**Pull request merged**